content/stellar-contracts/tokens/vault/vault.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Fungible Vault Token
+---
+
+[Source Code](https://github.com/OpenZeppelin/stellar-contracts/tree/main/packages/tokens/src/fungible/extensions/vault)
+
+The Fungible Vault Token implements the ERC-4626 tokenized vault standard,
```

Comment:
It's in the name but we can add explicitly that it extends and is compatible with "Fungible Token".